pypsa/clustering/spatial.py

```diff
@@ -385,6 +395,7 @@ def get_clustering_from_busmap(
     bus_strategies=dict(),
     one_port_strategies=dict(),
     generator_strategies=dict(),
+    aggregate_generator_buses=None,
```

Comment:
```suggestion
    aggregate_generators_buses=None,
```
to align with the other arguments